### PR TITLE
[Merged by Bors] - run examples on windows

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- ~~Running examples on Linux in CI timeout~~Linux is back!
- But hey we can run examples on windows too!

## Solution

- Run examples on windows daily
- I also added a 30 minutes timeout so that when it explodes, it doesn't explodes in 6 hours (the default timeout)
- And simplified the linux examples by not requiring a custom feature set

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (IceSentry) [inline .github/workflows/ci.yml:263]: Is there a reason for not using cache on windows?

**Comment by alice-i-cecile** [comment]
Added the `Critical` tag so we can merge things reliably. This LGTM (even if it is a bit of a hack), although I'm also curious about the cache.

**Review by alice-i-cecile — APPROVED** [review]

**Comment by mockersf** [comment]
I changed the PR to run examples on windows on a daily cron. It's still interesting to have the result, but not worth it to run it on every commit (and that way it'll be easy to copy/paste if there's a failure again)

File: .github/workflows/ci.yml
Comment (IceSentry) [reply .github/workflows/ci.yml]: I don't know if it's the case in the context of github runners, but as far as I know the paths in the linux run should be the same on windows. There's a `~` alias and windows supporst `/` so I think using the same paths should work.

**Review by IceSentry — APPROVED** [review]
LGTM

I wonder if there's other tests that could be moved to a daily only run to help make the PR pipeline faster. It's not a big issue and I guess at the speed we merge stuff it's better to make sure the PR is actually fully green.

**Comment by alice-i-cecile** [comment]
> I wonder if there's other tests that could be moved to a daily only run

IMO we should try to run all of the examples daily on all platforms. I don't have a strong preference about whether that's done in this PR or a follow-up.

**Comment by DJMcNab** [comment]
I suspect the examples should be run on the bors try/staging commits too, even if not run on every commit within a PR?

For example, being able to minimise windows on windows is both quite an important property to preserve, and apparently an easy property to lose. Without this test we can't ensure that PR authors maintain this property; only get a failure in the (as far as I know) low-visibility daily run.

**Comment by mockersf** [comment]
windows and iOS won't run on a commit/PR, but will run on a bors command

**Comment by mockersf** [comment]
bors try

**Comment by DJMcNab** [comment]
Hopefully this try should fail, since https://github.com/bevyengine/bevy/pull/3597 should catch https://github.com/bevyengine/bevy/pull/4527

Unless you haven't rebased I guess, but it looks like you have

**Comment by mockersf** [comment]
it failed on windows on the minimising example 🎉

**Comment by mockersf** [comment]
blocked on #4527

**Comment by mockersf** [comment]
bors try

**Pull request closed without merging**
